src/tools/rz-diff/binary_diffing.md

```diff
@@ -1,75 +1,181 @@
+       |----> classes (compares classes found in the files)
+       |----> command (compares command output returned when executed)
+       |----> entries (compares entries found in the files)
+       |----> fields (compares fields found in the files)
+       |----> graphs (compares 2 functions and outputs in graphviz/dot format)
+       |----> format (compares text files)
+       |----> imports (compares imports found in the files)
+       |----> libraries (compares libraries found in the files)
+       |----> sections (compares sections found in the files)
+       |----> strings (compares sections found in the files)
+       |----> symbols (compares symbols found in the files)
 
-$ rz-asm -d 85c00f94c0
-test eax, eax
-sete al
 ```
-Notice how the two jumps are nop'ed.
 
-For bulk processing, you may want to have a higher-level overview of differences. This is why rizin is able to compute the distance and the percentage of similarity between two files with the `-s` option:
+### Diffing ASCII-text files:
+
+
+rz-diff -t lines genuine cracked
 ```
-$ rz-diff -s /bin/true /bin/false
-similarity: 0.97
-distance: 743
+--- genuine
++++ cracked
+@@ -1,1 +1,1 @@
+-hello1234567890
++1234567890hello
 ```
 
-If you want more concrete data, it's also possible to count the differences, with the `-c` option:
+### Diffing functions in binaries:
+
+It this mode, it will give you three columns for all functions: "First file offset", "Percentage of matching" and "Second file offset".
+
+`rz-diff -t functions /bin/true /bin/false`
+
 ```
-$ rz-diff -c genuine cracked
-2
+         sym.imp.__fprintf_chk   11 0x00000000000024e0 | MATCH   (1.000000) | 0x00000000000024e0    11 sym.imp.__fprintf_chk
+               sym.imp.mbsinit   11 0x00000000000024f0 | MATCH   (1.000000) | 0x00000000000024f0    11 sym.imp.mbsinit
+              sym.imp.iswprint   11 0x0000000000002500 | MATCH   (1.000000) | 0x0000000000002500    11 sym.imp.iswprint
+         sym.imp.__ctype_b_loc   11 0x0000000000002510 | MATCH   (1.000000) | 0x0000000000002510    11 sym.imp.__ctype_b_loc
+                  fcn.00002640   34 0x0000000000002640 | UNMATCH (0.058824) | 0x0000000000002650    34 fcn.00002650
+                  fcn.00002700  840 0x0000000000002700 | UNMATCH (0.221163) | 0x0000000000002710   840 fcn.00002710
+                  fcn.00002b30  176 0x0000000000002b30 | UNMATCH (0.173077) | 0x0000000000002b40   176 fcn.00002b40
+                  fcn.00002bf0  208 0x0000000000002bf0 | SIMILAR (0.961538) | 0x0000000000002c00   208 fcn.00002c00
+                  fcn.00002cd0 4627 0x0000000000002cd0 | SIMILAR (0.993949) | 0x0000000000002ce0  4627 fcn.00002ce0
 ```
 
-If you are unsure whether you are dealing with similar binaries, with `-C` flag you can check there are matching functions. It this mode, it will give you three columns for all functions: "First file offset", "Percentage of matching" and "Second file offset".
+### Diffing classes in Binaries:
```

Comment:
I don't the the 'b' in 'binaries' needs to be capitalized. Can you get the others, too?
 
```suggestion
### Diffing classes in binaries:
```